docs/deployment/troubleshoot_deployment.md

```diff
@@ -1,33 +1,45 @@
+
+### Deploying the Cumulus Stack
 
-#### Configure Cumulus Stack
-##### vpc
+Monitoring the progress of stack deployment can be done from the [AWS CloudFormation Console](https://console.aws.amazon.com/cloudformation/home).
 
 Issues:
 
-  - If redeploying an existing configuration you may already have at least 1 vpc associated with your existing deployment, but its subnets can be transitory in nature depending on what kind of load balancing and/or docker activities are taking place at a given time.  You should  identify at least one persistent subnet to use as a subnet ID (you may only specify one) for use.    If this is needed, navigate to  [AWS EC2 > Auto Scaling Groups](https://console.aws.amazon.com/ec2/autoscaling/home) and note the "Availability Zone" (e.g., us-east-1a). Next, visit [AWS VPC](https://console.aws.amazon.com/vpc/home) and click on "Subnets". Copy the 'VPC' value into 'vpcId' and the appropriate 'Subnet ID' value, based on the Availability Zone value you just saw on the Auto Scaling Groups page, into 'subnets'. If you have no vpc and/or subnets, do not include the vpc section in your new configuration.
+#### **Error:** __"The availability zones of the specified subnets and the Auto Scaling group do not match"
 
-#### Deploy the Cumulus Stack
+See [vpc issues](#vpc)
 
-Monitoring the progress of stack deployment can be done from the [AWS CloudFormation Console](https://console.aws.amazon.com/cloudformation/home).
+#### Error: Stack.. is in ROLLBACK_COMPLETE (or ROLLBACK_FAILED) state and can not be updated.
 
-Issues:
+The stack cannot be re-deployed if it is currently in ROLLBACK_COMPLETE or ROLLBACK_FAILED.
+
+If this is a new deployment, delete the stack and try deploying again. 
+
+You may be able to continue the rollback operation. At the top of the CloudFormation page for the stack, click the 'Other Actions' dropdown and choose to continue rollback. 
+
+In the advanced settings when continuing rollback, you can enter the logical Ids of resources to skip that are preventing rollback. These ids can be found in the resources section of the CloudFormation page for the stack.
+
+#### Failure on nested stacks
 
--  **Error:** __"The availability zones of the specified subnets and the Auto Scaling group do not match"__ -- see [vpc issues](#vpc)
+If the deployment failed on nested stacks (CumulusApiDefaultNestedStack, CumulusApiV1NestedStack), and the nested stacks are gone due to rollback.  Try to deploy the main stack first by add nested_template parameter to your stack config app/config.yml file, and then run the deployment.
 
```

Comment:
Try to deploy the just the main stack first by adding a nested_template parameter set to null in your stack config app/config.yml file, and then run the deployment.